play.pokemonshowdown.com/src/panel-popups.tsx

```diff
@@ -308,8 +326,24 @@ class OptionsPanel extends PSRoomPanel {
 					class="trainersprite yours" width="40" height="40" style={{ verticalAlign: 'middle' }}
 					src={Dex.resolveAvatar(`${PS.user.avatar}`)}
 				/> {}
-				{PS.user.name}
+				<strong> {PS.user.name} </strong>
+			</p>
+			<p>
+				<button class="button" data-href="avatars"> Avatar..</button>
 			</p>
+
+			{
+				this.state.showStatusInput ?
+					<p>
+						<input name="statustext"></input>
+						<button class="button" onClick={this.editStatus}><i class="fa fa-pencil"></i></button>
+					</p> :
+					<p>
+						<button class="button" onClick={this.handleShowStatusInput} disabled={this.state.showStatusUpdated}>
+							{this.state.showStatusUpdated ? 'Status Updated' : 'Status..'}</button>
+					</p>
+			}
```

Comment:
```suggestion
			{this.state.showStatusInput ? (
					<p>
						<input name="statustext"></input>
						<button class="button" onClick={this.editStatus}><i class="fa fa-pencil"></i></button>
					</p>
				) : (
					<p>
						<button class="button" onClick={this.handleShowStatusInput} disabled={this.state.showStatusUpdated}>
							{this.state.showStatusUpdated ? 'Status Updated' : 'Status..'}</button>
					</p>
			)}
```

This is how I usually write ternaries in React for readability. It also lets you do `else ifs` by just `) : condition ? (`